src/user_workspaces_server/urls.py

```diff
@@ -65,18 +64,18 @@
     path("", parameter_view.ParameterView.as_view(), name="parameters"),
 ]
 
```

Comment:
Did you want to keep the commented-out code in this file in case we want to switch back?

Reply:
Yes, we can hot-fix more easily if we keep this around